projects/core/src/auth/user-auth/facade/auth.service.ts

```diff
@@ -87,10 +90,18 @@ export class AuthService {
    */
   async loginWithCredentials(userId: string, password: string): Promise<void> {
     try {
+      let uId: string = userId;
+
+      if (this.authMultisiteIsolationService?.getBaseSiteDecorator()) {
+        uId =
+          userId + this.authMultisiteIsolationService?.getBaseSiteDecorator();
+      }
+
```

Comment:
Do we _really_ need to wrap our whole new logic inside the `try catch` block? If not, let's move it up, outside of the block.

Reply:
I think is not necessary to have it in try/catch.